libs/stack/core-lib/src/utils/normalize-path.ts

```
@@ -0,0 +1,12 @@
+export function normalizePath(path: string): string {
+  const parts = path.split('/')
+
+  const trimmedParts = parts.reduceRight<string[]>((acc, part) => {
+    if (acc.length === 0 && part === '') {
+      return acc
+    }
+    return [part, ...acc]
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)